tests/test_stormevent.py

```diff
@@ -212,3 +215,25 @@ def test_storm_event_coops_product_within_region(florence2018):
 
     assert null_tidal_data["t"].sizes == {}
     assert east_coast_tidal_data.sizes == {"nos_id": 111, "t": 1}
+
+
+def test_status():
```

Comment:
Can we also mock a storm event so that realtime status can be tested as well?

Reply:
good idea, I wrote something for that in #47